packages/untp-test-suite/src/core/services/index.ts

```diff
@@ -1 +1,2 @@
 export * from './json-schema/index.js';
+export * from './dynamic-loading-schemas';
```

Comment:
The `untp-test-suite` is a module package, so I think we need to include '.js' at the end of the file import line.